src/components/send/components/AvailableToSend.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+
+import classNames from '../../../utils/classNames';
+import Accordion from '../../common/Accordion';
+import AccordionTrigger from './AccordionTrigger';
+import Breakdown from './Breakdown.css';
+import Amount from './entry_types/Amount';
+
+const translate = (type) => `sendV2.availableToSend.${type}`;
```

Comment:
What do you think about making this a bit more descriptive about what it's doing?

e.g. something like ```const prefixAvailableToSendId = (key) => `sendV2.availableToSend.${type}`;```